docs/about/index.rst

```diff
@@ -1 +1,4 @@
+📄 About

```

Comment:
Create extra chapter to avoid the long list of CHANGELOG entries in the main toc tree.